zng/time.go

```diff
@@ -55,6 +55,7 @@ func (t *TypeOfTime) ZSONOf(zv zcode.Bytes) string {
 	if err != nil {
 		return badZng(err, t, zv)
 	}
+	//XXX shouldn't this be UTC?
```

Comment:
It is, because `nano.Ts.Time` calls `time.Time.UTC`.

Reply:
(So you can zap this comment.)

Reply:
Okay, removing comment and removing the redunandant UTC() call in TypeOfTime.Marshal()